sylvia-derive/src/message.rs

```diff
@@ -415,6 +502,14 @@ impl<'a> ContractEnumMessage<'a> {
             MsgType::Query => quote! { #sylvia ::cw_schema::QueryResponses },
             _ => quote! {},
         };
+        let context_wrap = match msg_ty {
+            MsgType::Query => quote! {},
+            _ => quote! { ?; ctx.try_into().map_err(Into::into)},
+        };
+        let before_message = match msg_ty {
```

Comment:
Allows for executing code registered on the base (adapter for instance), before the module messages. This behavior is slightly different than the from_exec, because here it's the Impl here and not the trait.